pkg/cli/zip_per_node.go

```diff
@@ -529,17 +528,10 @@ func (zc *debugZipContext) collectPerNodeData(
 			}
 		} else {
 			s.done()
-			nodePrinter.info("%d ranges found", len(ranges.Ranges))
-			sort.Slice(ranges.Ranges, func(i, j int) bool {
-				return ranges.Ranges[i].State.Desc.RangeID <
-					ranges.Ranges[j].State.Desc.RangeID
-			})
-			for _, r := range ranges.Ranges {
-				s := nodePrinter.start("writing range %d", r.State.Desc.RangeID)
-				name := fmt.Sprintf("%s/ranges/%s", prefix, r.State.Desc.RangeID)
-				if err := zc.z.createJSON(s, name+".json", r); err != nil {
-					return err
-				}
+			s := nodePrinter.start("writing ranges")
+			name := fmt.Sprintf("%s/ranges.json", prefix)
+			if err := zc.z.createJSON(s, name, ranges.Ranges); err != nil {
```

Comment:
Not suggesting, just asking: Do we think we should marshal each individually, writing the results to a writer opened above the loop as we go, rather than marshaling all 100k at once? how big is one range json?